### ISSUE TITLE
write-json-file regression

### ISSUE THREAD

**Issue by edporras** [ngrunwald/datasplash#123]
@domparry bad news, I've ran into a problem with the output generated in that objects are getting stringified. ex., generating a pcoll of 

```edn
[{:id "23" :vals #{{:id "41" :views 342} {:id "52" :views 41}}}
 {:id "15" :vals #{{:id "44" :views 4} {:id "492" :views 59}}}]
```

results in this when calling `write-json-file`:

```
{"id":"23","vals":"#{{:id \"52\", :views 41} {:id \"41\", :views 342}}"}
{"id":"15","vals":"#{{:id \"492\", :views 59} {:id \"44\", :views 4}}"}
```

At first I was baffled how the test was not catching this and then I realized… I never added a write-json-file test 🤦‍♂️😭

I'll take a look at this tomorrow or over the weekend.

**Comment by domparry** [comment]
oh dear. I'll take a look at how cheshire handles sets.

**Comment by edporras** [comment]
I believe both Cheshire and data.json represent sets using vectors when writing. Should we check with Chris?

**Comment by domparry** [comment]
You are correct. I have created an issue for Chris here: https://github.com/cnuernber/charred/issues/6

**Comment by edporras** [comment]
@domparry released 0.7.9

**Comment by domparry** [comment]
Thabks @edporras!
